pkg/common/services/services.go

```diff
@@ -17,47 +28,28 @@ type Service struct {
 	InstanceID  string          `json:"instance_guid"`
 }
 
-type ObjectStoreCredentials struct {
-	AccessKeyID     string `json:"access_key_id" validate:"required"`
-	Bucket          string `json:"bucket" validate:"required"`
-	Host            string `json:"host" validate:"required"`
-	Region          string `json:"region" validate:"required"`
-	SecretAccessKey string `json:"secret_access_key" validate:"required"`
-	URI             string `json:"uri"`
-	Username        string `json:"username"`
-}
-
-// AMSCredentials are credentials from the legacy standalone authorization broker
-type AMSCredentials struct {
-	BundleURL   string                  `json:"bundle_url" validate:"required_without=ObjectStore"`
-	ObjectStore *ObjectStoreCredentials `json:"object_store" validate:"required_without=BundleURL"`
-	URL         string                  `json:"url" validate:"required"`
-	InstanceID  string                  `json:"instance_id"`
-}
-
-type MegacliteService struct {
-	Name        string `json:"name"`
-	Credentials struct {
-		URL string `json:"url"`
-	} `json:"credentials"`
-}
-
 type IASCredentials struct {
 	Certificate          string    `json:"certificate" validate:"required"`
 	CertificateExpiresAt time.Time `json:"certificate_expires_at"`
-	Clientid             string    `json:"clientid"`
+	ClientID             string    `json:"clientid"`
 	Domain               string    `json:"domain"`
 	Domains              []string  `json:"domains"`
 	Key                  string    `json:"key" validate:"required"`
 	OsbURL               string    `json:"osb_url"`
 	ProoftokenURL        string    `json:"prooftoken_url"`
 	URL                  string    `json:"url"`
 	ZoneUUID             string    `json:"zone_uuid"`
+
+	AmsInstanceID string `json:"authorization_instance_id"  validate:"required"`
+	AmsClientID   string `json:"authorization_client_id"`
+
+	// derived values
+	AmsServerURL        string
+	AmsBundleGatewayURL string
```

Comment:
make sure that JSONs don't interfere with derived values
```suggestion
	AmsServerURL        string    `json:"-"`
	AmsBundleGatewayURL string    `json:"-"`
```